packages/ui/src/components/molecules/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC, ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { cysyncLogoSmall } from '../../../assets/images';
+import { Flex, Image, Typography } from '../../atoms';
+import {
+  spacing,
+  SpacingProps,
+  width,
+  WidthProps,
+  height,
+  HeightProps,
+} from '../../utils';
+
+interface SideBarProps extends HeightProps, WidthProps, SpacingProps {
+  children?: ReactNode;
+}
+
+const SideBarStyle = styled.div<SideBarProps>`
+  display: flex;
+  height: 100%;
+  width: 100%;
+  flex-direction: column;
+  padding: ${({ theme }) => theme.spacing.five.spacing};
+  background: ${({ theme }) => theme.palette.background.sidebar};
+  ${width}
+  ${height}
+  ${spacing}
+`;
+
+export const SideBar: FC<SideBarProps> = ({ children, ...props }) => (
+  <SideBarStyle {...props}>
+    <Flex gap={8} mb={5}>
+      <Image src={cysyncLogoSmall} alt="cysynclogo" height={32} my="auto" />
+      <Typography variant="h3" color="muted" my="auto">
+        {' '}
+        cySync{' '}
```

Comment:
All the texts should come from the props in `packages/ui` components. And in `packages/core` you have to use `i18n` to get the texts.